README.md

```diff
@@ -1,2 +1,42 @@
+![logo](photo_points/wwwroot/images/fncf-rgb-horizontal-logo_1.jpg)
+
 # photo-points
-Web Application for Photo Points data and API
+Web Application for Photo Points data and API. 
+The web application, created for _Friends of North Creek Forest_, will allow the administrator to view approved photos, search photos, view pending photos from the mobile application. 
```

Comment:
The last part "view pending photos from the mobile application" is unclear. I think you may mean,  "... and view pending photos, submitted from the mobile app, for approval to post to the FNCF Photo Points website gallery.

Reply:
yes, that sounds a lot clearer, I've updated the change